backend/backend-infrastructure/ClassLibrary2/UnitOfWork.cs

```diff
@@ -0,0 +1,33 @@
+﻿using red_team.backend_infrastructure.Repositories;
+using red_team.Repositories.Interfaces;
+using System;
+
+namespace red_team.Repositories
+{
+    public class UnitOfWork : IDisposable
+    {
+        private SurveyContext db = new SurveyContext();
+        private SurveyRepository repository;
+ 
+        public SurveyRepository Surveys
+        {
+            get
+            {
+                if (repository == null)
+                    repository = new SurveyRepository(db);
```

Comment:
Code styles: code blocks always should be separated by Curly braces. Even if it's one line 
```csharp
                if (repository == null)
                {
                    repository = new SurveyRepository(db);
                }
```